docs/site/src/contributor/how/release-checklist.md

```diff
@@ -4,7 +4,7 @@
 
 ## Check additional automated tests
 - [ ] Verify that the latest E2E tests are fine (See "E2E *" workflows on https://github.com/cardano-foundation/cardano-wallet/actions).
-- [ ] Verify latest [buildkite nightly](https://buildkite.com/input-output-hk/cardano-wallet-nightly) and make sure the results are fine. [Benchmark charts](http://ec2-44-197-192-237.compute-1.amazonaws.com:5555) may be helpful in analysis.
+- [ ] Verify latest [buildkite nightly](https://buildkite.com/cardano-foundation/cardano-wallet-nightly) and make sure the results are fine. [Benchmark charts](http://ec2-44-197-192-237.compute-1.amazonaws.com:5555) may be helpful in analysis.
```

Comment:
The Buildkite organization is currently called `cardanofoundation`. I will change it to `cardano-foundation`, so that the link becomes correct.

Note: This change contradicts the pull request description. 🤓😈

Reply:
> Note: This change contradicts the pull request description.

Ah, thanks for spotting this!

My preference would have been to exclude this change from the PR, since as you point out, it's not within this PR's scope.

But given that you're happy to change the Buildkite organisation name from `cardanofoundation` to `cardano-foundation`, I'm happy to leave this change in!

Reply:
Oops. Turns out that I cannot change the name because there already exists a Buildkite organization `cardano-foundation`. 🙈

Reply:
Fixed.